src/pages/Main.js

```diff
@@ -1,13 +1,349 @@
 import React from 'react';
+import Card from './CardComponents/Card';
 import styled from 'styled-components';
 
 const Main = () => {
-  return <Div>돌돌</Div>;
+  return (
+    <MainLayout>
+      <MainBox>
+        <MainWrap>
+          <MainContent>
+            <ContentText>
+              위코드에서의 마지막 추억을 돌하룻밤에서 보내세요. <br />
+              모두 고생하셨습니다 또 만나요.
+            </ContentText>
+            <ContentBtn>자세히 알아보기</ContentBtn>
+          </MainContent>
+          <MainImg
+            src="https://images.unsplash.com/photo-1478131143081-80f7f84ca84d?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1170&q=80"
+            alt="main"
+          />
+          <MainText>호기심을 자극하는 제주로 떠나보세요</MainText>
+          <MainBtn>
+            <div>유연한 검색</div>
+          </MainBtn>
+        </MainWrap>
+        <CardImgTit>설레는 다음 여행을 위한 아이디어</CardImgTit>
+        <FlexBox>
+          {CARD_LIST.map(cardInfo => {
+            return (
+              <Card
+                key={cardInfo.id}
+                id={cardInfo.id}
+                name={cardInfo.name}
+                distance={cardInfo.distance}
+                imageUrl={cardInfo.imageUrl}
+              />
+            );
+          })}
```

Comment:
파라미터 자리에서 구조 분해 할당 할 수 있습니다.
```suggestion
    
  {
    CARD_LIST.map(({ id, name, distance, imageUrl }) => {
      return (
        <Card
          key={id}
          id={id}
          name={name}
          distance={distance}
          imageUrl={imageUrl}
        />
      );
    });
  }
```

Reply:
가독성이 더 좋아졌습니다! 감사합니다 유진님~